tests/docker/test_container_manager.py

```diff
@@ -67,12 +60,11 @@ def test_containers_for_image_client_api_without_user(self):
 
         # We assume the client.containers(filters=...) is tested by docker-py
         # Instead we test if the correct arguments are passed to the Client API
-        yield self.manager.containers_for_image("imageid")
+        yield self.manager.container_from_mapping_id("user", "mapping")
         call_args = self.manager.docker_client.client.containers.call_args
 
         # filters is one of the keyword argument
         self.assertIn('filters', call_args[1])
-        self.assertEqual(call_args[1]['filters']['ancestor'], "imageid")
 
     @gen_test
     def test_containers_for_image_client_api_with_user(self):
```

Comment:
Not exactly.  In the test above we already mocked the return results of `docker_client.containers` to be a list of valid dictionary containing containers' info, the `filters` does not matter in that test.  We still need to check if the docker client API is utilised as expected.